cli/src/run_testenv.rs

```diff
@@ -39,6 +39,9 @@ pub struct RunTestenv {
     /// When this process stops, the server will be automatically killed
     #[arg(long)]
     main_process_id: u32,
+    /// Skip initialization
+    #[arg(short, long)]
+    empty: Option<bool>,
```

Comment:
You can configure `clap` to use a default value
 
```suggestion
    /// Skip initialization
    #[arg(short, long, default_value_t)]
    empty: bool,
```

The code above will set `empty` to `false` when the value wasn't provided in the CLI

https://docs.rs/clap/latest/clap/_derive/index.html#arg-attributes

Reply:
Nice